pages/features.js

```diff
@@ -11,13 +15,10 @@ import Hero from '../src/Hero'
 import FeaturesHero from '../src/FeaturesHero'
 import TrySection from '../src/TrySection'
 
-const HeadInjector = () => (
```

Comment:
why do we move it down?

Reply:
I moved it under the main component, `FeaturesPage`, like the rest of the private components in this module. Also reorganized them into the order they appear in the JSX returned by `FeaturesPage` (and moved the `export` statement to the end of the file.)

Reply:
kk, was asking if there was a specific reason for this (linter complaining for example). Not sure what is the best way (if it's important at all) to do this. I would check some style guides from Facebook or Google (?) or some other big techs.

Reply:
Actually yes this was prompted by the linter demanding a name for the export, same as in https://github.com/iterative/dvc.org/pull/776#pullrequestreview-312085056. 🙂

The Airbnb React/JSX style guide seems interesting. I think in this case I'll switch to direct `export` statements ([example](https://github.com/airbnb/javascript/tree/master/react#declaration)), as also confirmed by Alexey in https://github.com/iterative/dvc.org/pull/776#pullrequestreview-312006609 above...

Reply:
Changing same as https://github.com/iterative/dvc.org/pull/776#discussion_r343470755.